package.json

```diff
@@ -17,7 +18,7 @@
     "test:notification:production": "node build/test/functional/test-notification.js"
   },
   "engines": {
-    "node": ">=12.0.0 <15.0.0"
+    "node": ">=12.0.0 <15.11.0"
```

Comment:
I understand that there could be good reasons for keeping `<15.0.0`, which I don't know.

Reply:
I was keep it below 15 only because some folks were having problems with `tsc`. We should probably get on latest sooner than later though!